### PR TITLE
adding masterdata import role

### PR DESCRIPTION
**Pull request by conckja1**
adding masterdata import role

### CODE REVIEW

File: src/server/db/migrations/050.1-assignableRole.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    {
+        'ownedRoleId': 'admin',
+        'assignableRoleId': 'invoice-masterdata-import',
+        'createdBy': 'The Doctor'
+    }
+];
+
+module.exports.up = async function(db, config)
+{
+    assignableRoles.forEach(assignableRole => assignableRole.createdOn = new Date());
```
Comment (denic) [inline src/server/db/migrations/050.1-assignableRole.test.js]: Y are u using a single item array here and all the array stuff + bulk insert to add a single item?

File: src/server/db/migrations/050.2-userRole.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+const roles = require('../data/userRoles-30.json');
```
Comment (denic) [inline src/server/db/migrations/050.2-userRole.test.js]: Is this different from the AssignableRoles above in that sense that you are using a JSON here and above an inline decleration?

File: src/server/db/migrations/050.1-assignableRole.test.js (hunk 1 above)
Comment (conckja1) [reply src/server/db/migrations/050.1-assignableRole.test.js]: yes, I conveniently copy-pasted previous code. There is no performance penalty from doing so.

File: src/server/db/migrations/050.2-userRole.test.js (hunk 2 above)
Comment (conckja1) [reply src/server/db/migrations/050.2-userRole.test.js]: there is no special reason for that.

**Pull request merged**